documentation/asciidoc/topics/yaml/publisher_labels_downstream.yaml

```diff
@@ -1,13 +1,3 @@
-# Defines global labels for services.
```

Comment:
Can we leave a comment that `INFINISPAN_OPERATOR_TARGET_LABELS` is also available for services though we don't use it atm?

Reply:
@rigazilla Sure but we should expose `INFINISPAN_OPERATOR_TARGET_LABELS` in community docs only, right? Or is it also possible to have this downstream but it's just ignored at the moment?

Reply:
@oraNod it's unused atm, but it's possible to have it downstream also.